Dockerfile

```diff
@@ -222,7 +222,7 @@ RUN --mount=type=cache,target=/root/.cache/pip/,id=pip-cache \
     && rm --recursive --force --verbose /var/tmp/* \
     && rm --recursive --force --verbose /var/cache/apt/archives/* \
     && truncate --size 0 /var/log/*log
-
+RUN  --mount=type=cache,target=/root/.cache/pip/,id=pip-cache python3 -m pip install --default-timeout=1000 django-auditlog 
```

Comment:
This is not the right way to add a new dependency

Reply:
Eh...I am stupid. Thats the way I did it at the begining as I am not familiar with the other systems being used and couldn't figure it out quickly.  What is the exact system being used for the requirements?

Reply:
We use [`pipenv`](https://pipenv.pypa.io/en/latest/) for Python dependency management.  Running `pipenv upgrade <package name> will add a new package to the Pipfile and Pipfile.lock.  The Dockerfile will handle it from there.

Reply:
So I think I have manged this......